website/static/js/fangorn.js

```diff
@@ -2090,13 +2100,15 @@ tbOptions = {
     onmouseoverrow : _fangornMouseOverRow,
     sortDepth : 2,
     dropzone : {                                           // All dropzone options.
+        maxFilesize: 10000000,
```

Comment:
Waterbutler does have a max file limit but fangorn also checks file size in other parts of the code.